### ISSUE TITLE
nanoc-git seems not work in nanoc

### ISSUE THREAD

**Issue by stardiviner** [nanoc-archive/nanoc-git#2]
I followed the README instruction, add config section into `nanoc.yml`.
and execute command `$ nanoc deploy`.
and have branch `gh-pages`.
but command error output:

```
$ nanoc deploy
Loading site data… done
Error: The specified deploy target has an unrecognised kind “git” (expected one of fog, rsync).
```

Do you know what I missed?

**Comment by TheLonelyGhost** [comment]
I tried what nanoc/nanoc-lftp#2 suggested by adding a require line to `lib/default.rb`, but that didn't work. Looks like the latest version in rubygems is '0.0.7' whereas this one is '1.0.0'.

What I did to get it to work was use bundler to checkout the master branch from git like so:

``` ruby
source "https://rubygems.org"

gem 'nanoc'
gem 'adsf'
gem 'nanoc-git', git: 'https://github.com/nanoc/nanoc.git'
```

Run your usual `bundle install` command before executing `bundle exec nanoc deploy -D` and that should be a temporary fix.

After all of this, running `bundle exec nanoc deploy -D` revealed 3 deployment strategies instead of the default 2. All in all, looks like we're waiting on a release of 1.0.0 to rubygems in order to fix this in a more robust way.

**Comment by lifepillar** [comment]
Thanks for reporting that the gem needs to be updated. I'll try to do it soon.

**Comment by stardiviner** [comment]
@lifepillar Thanks

**Comment by lifepillar** [comment]
Nanoc-git has been updated to support Nanoc 4. To use it right now, put this in your `Gemfile`:

```
gem 'nanoc-git',
    :git => 'git://github.com/nanoc/nanoc-git.git',
    :branch => 'master'
```

The gem will be hopefully published soon. Feel free to open another issue if something does not work as expected.
